backend/tests/events.test.ts

```diff
@@ -92,3 +93,133 @@ describe ("Testing DELETE event",() => {
+    const response = await request(app).get("/events/" + eventid + "/attendees");
+    expect(response.status).toBe(200);
+  })
+})
+
+  test("Get attendees for invalid event", async () => {
+
+    const eventid = -1;
+    const response = await request(app).get("/events/" + eventid + "/attendees");
+    expect(response.body.length).toBe(0);
+  })
+
+  describe ("Testing POST/events/:eventid/:attendeeid", () => {
+    test("Add attendee for existing event", async () => {
+
+      const events = await request(app).get("/events");
+      const users = await request(app).get("/users");
+      const eventID = events.body[1].id;
+      const attendeeid = users.body[1].id;
+      const response = await request(app).post("/events/" + eventID + "/"+ attendeeid);
+      expect(response.status).toBe(200);
+    })
+  })
+
+  describe("Testing DELETE /events/:eventid/attendees/:attendeeid", () => {
+    test("Delete an attendee", async () => {
+      const events = await request(app).get("/events");
+      const attendees = await request(app).get("/users");
+      const eventid = events.body[1].id;
+      const attendeeid = attendees.body[0].id;
+      const response = await request(app).delete("/events/" + eventid + "/attendees/" + attendeeid);
+      expect(response.status).toBe(200);
+    });
+
+    test("Delete an invalid attendee", async () => {
+      const events = await request(app).get("/events");
+      const eventid = events.body[1].id;
+      const attendeeid = events.body[0].userID;
+
+      const response = await request(app).delete("/events/" + eventid + "/attendees/" + attendeeid);
+      expect(response.status).toBe(500);
+    });
+  });
+
+  describe("Testing PATCH /events/:eventid/status/:status", () => {
+    test("Update event status to active", async () => {
+      const events = await request(app).get("/events");
+      const eventid = events.body[0].id;
+      const status = "ACTIVE";
+      const response = await request(app).patch(`/events/${eventid}/status/${status}`)
+      const data = response.body
+      expect(response.status).toBe(200);
+      expect(data.status).toBe("ACTIVE"); 
+    });
+    
+    test("Event status invalid", async () => {
+      const events = await request(app).get("/events");
+      const eventid = events.body[0].id;
+      const status = "COMPLETE";
+      const response = await request(app).patch("/events/" + eventid + "/status/" + status);
+      expect(response.status).toBe(500);
+    });
+  });
+
+  describe("Testing PATCH /events/:eventid/owner/:ownerid", () => {
+    test("Change current owner", async () => {
+      const events = await request(app).get("/events");
+      const eventid = events.body[1].id;
+      const users = await request(app).get("/users");
+      const ownerid = users.body[0].id;
+      const response = await request(app).patch("/events/" + eventid + "/owner/" + ownerid); 
+      const data = response.body;
+      expect(data.ownerid).toBe("clfw74ia70000jsacl558b0ub");
```

Comment:
You can't hardcode a an eventID like this. The test script generates a unique set of users and events everytime it is run so this will always fail